src/app/app-routing.module.ts

```diff
@@ -11,7 +11,12 @@ import {UserActionsComponent} from '@app/components/homepage/user-actions/user-a
 import {NotFoundComponent} from '@app/components/general/not-found/not-found.component'
 import {ForbiddenComponent} from '@app/components/general/forbidden/forbidden.component'
 import {AdminComponent} from "@app/admin/admin.component"
+import {UserStatsComponent} from "@app/components/user-stats/user-stats.component"
 
+const routerOptions: ExtraOptions = {
```

Comment:
We don't need this to be a separate variable, add it inline in the router module.